private/data/i18n/en.json

```diff
@@ -429,7 +429,15 @@
         "registerGuest": "Hello! Add an email and receive order updates.",
         "trackYourDelivery": "Order updates will be sent to",
         "noOrdersFound": "No orders found.",
-        "adminMessage": "You are viewing orders as an administrator"
+        "adminMessage": "You are viewing orders as an administrator",
```

Comment:
Let's go ahead and add the LingoHub translations in this PR as well.. don't you think?